app/package.json

```diff
@@ -48,7 +42,8 @@
     "fetch-readablestream": "^0.2.0",
     "mocha": "^5.0.1",
     "mocha-headless-chrome": "^2.0.3",
-    "sinon": "^7.3.2"
+    "sinon": "^7.3.2",
+    "webpack": "^4.1.1"
```

Comment:
do we need webpack in both package.json files?

Reply:
ah sorry I just noticed the TODO comment - still curious why do we need the two versions now

Reply:
It's because the `test` script of `app/package.json` uses it.
I could also move the webpack config that the `test` script refers to into the root folder, though then I would also need to change some mocha dependencies. Maybe it is the right thing to do, though I thought that it may be a little out of the scope of this PR. What do you think? Should I move it to the root and also move some mocha dependencies?

Reply:
I'd do it now if it isn't too much effort, better have it in a consistent state to not bloat our backlog with further "tech debt", especially if it is little effort now, as doing it later would just incur the cost in form of time spent onboarding/sorting out webpack inconsitencies that possibly emerged after this PR